app/src/examples/PinExample.tsx

```diff
@@ -53,14 +53,21 @@ function NumberDisplay({ number }: { number: Animated.SharedValue<number> }) {
     <View style={styles.numberContainerOuter}>
       <View style={styles.numberContainerInner}>
         {indices.map((i) => {
-          return <Digit digit={getDigit(number, i)} key={i} />;
+          return <Digit number={number} index={i} key={i} />;
         })}
       </View>
     </View>
   );
 }
 
-function Digit({ digit }: { digit: Animated.SharedValue<number> }) {
+function Digit({
+  number,
+  index,
+}: {
+  number: Animated.SharedValue<number>;
+  index: number;
+}) {
```

Comment:
Move to `DigitProps`?

Reply:
[ab7a8a0](https://github.com/software-mansion/react-native-reanimated/pull/5361/commits/ab7a8a01963612cf2a92ef6d94c1224ecdb90403)